src/transactions/TransactionUtils.cpp

```diff
@@ -847,16 +861,24 @@ setAuthorized(LedgerTxnHeader const& header, LedgerTxnEntry& entry,
 bool
 trustLineFlagIsValid(uint32_t flag, uint32_t ledgerVersion)
 {
+    uint32_t invalidAuthCombo =
```

Comment:
I would prefer to have this in the else (for protocol 13 and later) as this doesn't apply for older versions of the protocol

Reply:
Done